Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/CreateAndQueryChangeRequestV2/task.json

```diff
@@ -290,6 +295,43 @@
       "groupName": "completionOptions",
       "helpMarkDown": "Type complex expression for success criteria.",
       "visibleRule": "successCriteria = advanced"
+    },
+    {
+      "name": "failureCriteria",
+      "type": "pickList",
+      "label": "Failure criteria",
+      "defaultValue": "desiredStatus",
+      "options": {
+        "advanced": "Advanced failure criteria",
+        "desiredStatus": "Desired status of change request"
+      },
+      "required": "false",
+      "helpMarkDown": "Choose failure criteria for change request.",
+      "properties": {
+         "EditableOptions": "false"
+      },
+      "groupName": "rejectionOptions"
+    },
+    {
+      "name": "DesiredFailureExitStatus",
+      "type": "pickList",
+      "label": "Desired status of change request",
+      "required": "true",
+      "groupName": "rejectionOptions",
+      "helpMarkDown": "Choose or type status of change request that indicates the change request can be rejected. Gate would fail immediately when the the change request status is same as the provided value.",
+      "properties": {
+        "EditableOptions": "True"
+      },
+      "visibleRule": "failureCriteria = desiredStatus"
+    },
+    {
+      "name": "AdvancedFailureCriteria",
```

Comment:
This one is tested for various expressions ?

Reply:
Yes, resolved some product doubts with @shashban, will be testing all the scenarios after the changes.